### DISCUSSION TITLE
generate  to one dart file ?

### DISCUSSION THREAD

**Discussion by postflow** [hiranthaR/Json-to-Dart-Model#33] [Q&A]
Hi! How to generate all nested classes to one dart file ? (One big dart file instead some small files in one catalog)

**Comment by iamarnas** [comment]
@postflow Hi! 
It is not supported with Json to Dart Model generator because separated files needs for support `Json Serializable` and `Freezed` classes and also provide better support for duplicates objects.  No need to rename objects with fluffy names for keep the code cleaner and more readable. Therefore, this generator is unique from others. And in reality a one big `Dart` file or few saparated files it gives the same code length. For now I have more important things. Maybe in the future will be possible generate to single file If users will require it but in my opinion separated files more professionally.

**Comment by iamarnas** [comment]
Some tips for beginners how to keep cleaner imports: use [barrels](https://marketplace.visualstudio.com/items?itemName=miquelddg.dart-barrel-file-generator).
